### DISCUSSION TITLE
Specifying field exclusion

### DISCUSSION THREAD

**Discussion by Lioness100** [mikro-orm/mikro-orm#2984] [Q&A]
If I had an entity with a lot of properties, and wanted to select all of them in `fields` except for one, is there an easier way than specifying all the other properties? For example:

```ts
// The only field I want to exclude is `ten`
repository.find({ ... }, { fields: ['one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine'] });
```

Is very verbose, and hard to maintain when properties are added, removed, or renamed. I'm using MongoDB, and in the raw driver you could [specify inclusion/exclusion with numbers](https://www.mongodb.com/docs/manual/reference/method/db.collection.find/#explicitly-excluded-fields). Is there any way to reproduce something like that with MikroORM?

**Comment by Lioness100** [comment]
Hi @B4nan, I hate to bother you, but is there any way you could help me out?

**Comment by Lioness100** [comment]
Hi @B4nan, sorry to bother you again, but did you get a chance to look at my question?

**Comment by Lioness100** [comment]
Any updates?

**Reply by robertanton81** [reply]
maybe   @Property({ hidden: true }) when you define your entity ?

**Reply by Lioness100** [reply]
I don't think that really solves the problem

**Reply by B4nan** [reply]
For exclusion its `lazy: true`, but I guess you already know about that. There is no `excludes` support currently in `FindOptions`, PR welcome, it should be rather simple to support.

**Reply by Lioness100** [reply]
What would be the best strategy to add in support? I'm not really familiar with the codebase.
